episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,402 @@
+we are looking to quantify many nuclei at many different timepoints, it's not 
+feasible to do this manually - we'll need to develop an automated workflow to 
+segment the nuclei and measure their volume and sphericity. We'll look at some 
+techniques for this kind of analysis in the [manual segmentation](FIXME.md), 
+[thresholding](FIXME.md) and [instance segmentation](FIXME.md) episodes.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Choose a light microscopy method
+
+For the final part of this episode, let's look at how we can choose a light 
+microscopy method for our experiment. We'll look at the last step (choosing 
+acquisition settings) in the next episode.
+
+Often the first step in choosing a light microscopy technique is discussing with 
+other members of your research group or department. What are most of them using, 
+for what kind of experiments? If you are working with a microscopy core facility, 
+then the people who work there are also a great source of advice and 
+recommendations. Finally, you can also explore the literature for research 
+questions and experiments similar to yours to see which methods are used most 
+often.
+
+There's a very wide variety of light microscopes available (from many different 
+manufacturers) that are specialised for different kinds of samples. We'll take a 
+brief look at some popular options below, but bear in mind that there are many 
+more!
+
+### Widefield
+
+In a widefield microscope, the entire sample is illuminated by the light source 
+at once. The light source can illuminate the sample from below (in an upright 
+microscope) or from above (in an inverted microscope).
+
+In it's simplest form, a widefield microscope can be used for 'brightfield 
+microscopy'. This is simply where the sample is illuminated by a bright light 
+from one side, and then imaged from the other. The issue with this method is 
+that it often produces low-contrast images, where biological structures are 
+hard to see. This is because biological structures are often quite transparent - 
+they don't absorb much light or differ much in their density. For this reason, 
+contrast agents/staining are often used to increase contrast (i.e. the addition 
+of dyes/chemicals that bind to specific structures).
+
+![The image above is Napari's Skin (RGB) sample image - it is a brightfield 
+image of a hematoxylin and eosin stained slide of dermis and epidermis.
+](fig/skin.png){alt="Screenshot of Napari's Skin sample image" width='70%'}
+
+To increase contrast (especially for unstained samples), widefield microscopes 
+often support ['phase-contrast'
+](https://www.microscopyu.com/techniques/phase-contrast/introduction-to-phase-contrast-microscopy) 
+or ['DIC - Differential Interference Contrast'
+](https://www.leica-microsystems.com/science-lab/microscopy-basics/differential-interference-contrast-dic/).
+
```

Comment:
Is it worth one sentence explaining what these techniques do to improve contrast?